Core/Parsing/ExpressionVisitors/TreeEvaluation/EvaluatableTreeFindingExpressionVisitor.cs

```diff
@@ -541,7 +544,7 @@ protected override Expression VisitRelinqUnknownNonExtension (Expression express
     /// <see cref="ExpressionVisitor"/> has a dedicated Visit method for it; otherwise, <see langword="false"/>. 
     /// Note that <see cref="ExpressionType.Extension"/>-type expressions are considered 'not supported' and will also return <see langword="false"/>.
     /// </returns>
-    private bool IsCurrentExpressionEvaluatable (Expression expression)
+    private static bool IsCurrentExpressionEvaluatable (Expression expression)
```

Comment:
Did you change the method to static because of performance reasons? I know there's a small difference in execution speed between calling instance methods and static methods but is it actually measurable?